src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/UpdateManyCommand.java

```diff
@@ -27,5 +29,13 @@ public record Options(
               description =
                   "When `true`, if no documents match the `filter` clause the command will create a new _empty_ document and apply the `update` clause and all equality filters to the empty document.",
               defaultValue = "false")
-          boolean upsert) {}
+          boolean upsert,
+      @Nullable
+          @Schema(
+              description = "Next page state for pagination.",
+              type = SchemaType.STRING,
+              implementation = String.class)
+          @JsonProperty("pageState")
+          @JsonAlias("pagingState") // old name, 1.0.0-BETA-3 and prior
```

Comment:
Do we need this also here? I know it is used for older cases. I guess that makes sense for consistency, and we need not document it (i.e. only include main new option in descriptions).

Reply:
Just used what's available in `find` command.

Reply:
Right, figured. That's fine.